source/base/DateTime.ooc

```diff
@@ -274,4 +272,19 @@ DateTime: cover {
 	dateIsValid: static func (year, month, day: Int) -> Bool {
 		year >= 1 && month in?(1 .. 13) && day in?(1 .. This daysInMonth(year, month) + 1)
 	}
+	kean_base_dateTime_getTicks: unmangled func -> UInt64 { this _ticks }
+	kean_base_dateTime_new: unmangled static func (ticks: UInt64) -> This { This new(ticks) }
+	kean_base_dateTime_fromDate: unmangled static func (year, month, day: Int) -> This { This new(year, month, day) }
+	kean_base_dateTime_fromTime: unmangled static func (hour, minute, second, millisecond: Int) -> This { This new(hour, minute, second, millisecond) }
+	kean_base_dateTime_fromDateTime: unmangled static func (year, month, day, hour, minute, second, millisecond: Int) -> This {
+		This new(year, month, day, hour, minute, second, millisecond)
+	}
+	kean_base_dateTime_now: unmangled static func -> This { This now }
+	kean_base_dateTime_millisecond: unmangled func -> Int { this millisecond() }
+	kean_base_dateTime_second: unmangled func -> Int { this second() }
+	kean_base_dateTime_minute: unmangled func -> Int { this minute() }
+	kean_base_dateTime_hour: unmangled func -> Int { this hour() }
+	kean_base_dateTime_day: unmangled func -> Int { this day() }
+	kean_base_dateTime_month: unmangled func -> Int { this month() }
+	kean_base_dateTime_year: unmangled func -> Int { this year() }
```

Comment:
Probably because this was my first task and my background is C++. There are no properties there (like in ooc), so I just used functions like I always did (with the difference that you can't mark a function as `const` in ooc) :)